regulations/static/regulations/css/scss/_typography.scss

```diff
@@ -178,13 +178,13 @@ a:visited {
 .internal:visited {
   color: $black;
   text-decoration: none;
-  border-bottom: 1px solid $blue_light;
+  border-bottom: 1px solid $action_color;
 }
 
 .internal:hover,
 .internal:active,
 .internal:focus {
-  background-color: $blue_lighter;
+  background-color: $action_color_dark;
```

Comment:
Good call. I think I'd rather just scrap this background color thing in favor of making the text and bottom border a bit darker, following the pattern of how we do other types of hover styles on the site as opposed to adding a background where one didn't exist before.